source/material_model/interface.cc

```diff
@@ -747,6 +747,46 @@ namespace aspect
       return vs;
     }
 
+    namespace
+    {
+      std::vector<std::string> make_plastic_additional_outputs_names()
+      {
+        std::vector<std::string> names;
+        names.push_back("weakened_cohesions");
+        names.push_back("weakened_friction_angles");
```

Comment:
I agree with @gassmoeller here. While the original intent was only to do strain weakening, it would be good to leave the door open for implementing strain hardening and not having to rename these terms. Another term that could be used is `modified...`.